To Do App/script.js

```diff
@@ -2,18 +2,88 @@ const form = document.querySelector("form");
 const input = document.querySelector("#txtTaskName");
 const btnDeleteAll = document.querySelector("#btnDeleteAll");
 const taskList = document.querySelector("#task-list");
+// const items = ["item 1", "item 2", "item 3", "item 4"]; (veri tabanından gelen değerleri yansıtmak için yaoruma alındı)
+let items;
+
+
+// Load items
+loadItems();
+
 //Call Event Listener
 eventLiteners();
 function eventLiteners() {
     // submit event
     form.addEventListener("submit", addNewItem);
+
+    // delete item event
+    taskList.addEventListener("click", deleteItem);
+
+    // delete all item event
+    btnDeleteAll.addEventListener("click", deleteAllItems);
+};
+
+function loadItems() {
```

Comment:
Bu fonksiyon normalde async olarak yazilir. Erken oldugundan girmemistir videoda.

Reply:
```suggestion
async function loadItems() {
items = await getItemsFromLS();
items.forEach(function (item) {
createItem(item);
})
};
```

Normalde js her satiri ayni anda calistirir yani senkron calisir. Async dedigimizde asenkron calisir. Yani önce await ile belirttigin satiri bekler, onun isi bitince digerlerine devam eder. 
Burada getItemsFromLS() veritabani islemini simüle etmek icin kullaniliyor. Normalde veritabani islemi hemen bitmeyeceginden o satiri beklemediginde js hata verir. Bu sebeple cok hata oluyor ve baya ugrasiyoruz. 
Bu kavramlar gelecek karsina. Simdiden medium dan yazilar okuyabilirsin. Anlayip rahat kullanabilmek biraz sürüyor

Reply:
woowww olumlu.